contrib/yolov5/rikai/contrib/yolov5/model.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+import torch
+import torch.nn as nn
+from torch.cuda import amp
+from yolov5.models.common import Detections
+from yolov5.utils.datasets import letterbox
+from yolov5.utils.general import (
+    make_divisible,
+    non_max_suppression,
+    scale_coords,
+)
+from yolov5.utils.torch_utils import time_sync
+
+from rikai.contrib.torch import RikaiModule
+
+
+class RikaiYolov5Model(nn.Module, RikaiModule):
```

Comment:
Lets just rename it to `Yolov5`?

And how does this work with traning, deployment and etc?

Reply:
> Lets just rename it to Yolov5?

Renamed.

> And how does this work with traning, deployment and etc?

This interface adds an extra parameter: option, in this way, we could pass more options via

``` sql
create or replace model xxx
options(
 op1='x',
 op2=0.3
)
```